### ISSUE TITLE
Monitor fails to start when no Hazelcast exporter is present

### ISSUE THREAD

**Issue by Zelldon** [zeebe-io/zeebe-simple-monitor#66]
I tried to start the Simple monitor without the Hazelcast Exporter in the Zeebe Broker and this brakes.

After adding the exporter to the Broker everything works fine.

Tries this on Windows.

**Comment by saig0** [comment]
Since 0.13.0, the simple monitor requires the Hazelcast exporter. However, we can make it optionally so that the monitor still works without it. Only the push notifications don't work in this case.

**Comment by jwulf** [comment]
What is the error message in this scenario?

**Comment by bkc** [comment]
Hi, 

this issue is closed, but the  0.15.0 docker image fails to run when hazelcast is not available.

I even tried setting env io.zeebe.monitor.hazelcast.connection='' (or empty), it still tries to connect.

Should 0.15.0 work w/o hazelcast?  

```
2019-07-26 02:57:00.954  INFO 1 --- [           main] i.z.m.zeebe.ZeebeConnectionService       : Connecting to broker '10.152.183.26:26500'
2019-07-26 02:57:01.756  INFO 1 --- [           main] i.z.m.zeebe.ZeebeConnectionService       : connected to '10.152.183.26:26500'
2019-07-26 02:57:01.757  INFO 1 --- [           main] i.z.m.zeebe.ZeebeConnectionService       : connected to '10.152.183.26:26500'
2019-07-26 02:57:01.770  INFO 1 --- [           main] i.z.m.zeebe.ZeebeConnectionService       : Connecting to Hazelcast 'localhost:5701'
2019-07-26 02:57:02.208  WARN 1 --- [ient_0.cluster-] c.h.c.c.ClientConnectionManager          : hz.client_0 [dev] [3.11.1] Exception during initial connection to [localhost]:5701, exception
 com.hazelcast.core.HazelcastException: java.net.SocketException: Connection refused to address localhost/127.0.0.1:5701
2019-07-26 02:57:02.208  WARN 1 --- [ient_0.cluster-] c.h.c.c.ClientConnectionManager          : hz.client_0 [dev] [3.11.1] Unable to get alive cluster connection, try in 3000 ms later, atte
mpt 1 of 2.
2019-07-26 02:57:05.209  WARN 1 --- [ient_0.cluster-] c.h.c.c.ClientConnectionManager          : hz.client_0 [dev] [3.11.1] Exception during initial connection to [localhost]:5701, exception
 com.hazelcast.core.HazelcastException: java.net.SocketException: Connection refused to address localhost/127.0.0.1:5701
2019-07-26 02:57:05.210  WARN 1 --- [ient_0.cluster-] c.h.c.c.ClientConnectionManager          : hz.client_0 [dev] [3.11.1] Unable to get alive cluster connection, attempt 2 of 2.
2019-07-26 02:57:05.211  WARN 1 --- [ient_0.cluster-] c.h.c.c.ClientConnectionManager          : hz.client_0 [dev] [3.11.1] Could not connect to cluster, shutting down the client. Unable to 
connect to any address! The following addresses were tried: [[localhost]:5701]
```

**Comment by saig0** [comment]
Since the version 0.15.0, the simple monitor uses the Hazelcast exporter to import the data from Zeebe. So, if there is no Hazelcast running, no data can be imported. In order to not confuse users (e.g. "I deployed workflows successfully but they are not shown in the list..?"), I decided to fail.
However, it only tries to connect to Hazelcast when it is connected to a Zeebe broker.

Does this make sense for you?
